inspire_dojson/cds/rules.py

```diff
@@ -397,15 +397,19 @@ def keywords(self, key, value):
 
 
 @cds2hep_marc.over('693__', '^693..')
+@utils.flatten
 @utils.for_each_value
 def accelerator_experiments(self, key, value):
     accelerator = ignore_not_applicable(value.get('a'))
-    experiment = ignore_not_applicable(value.get('e'))
-    experiment = EXPERIMENTS.get((accelerator, experiment), experiment)
-    return {
-        'a': accelerator,
-        'e': experiment,
-    }
+    experiments = force_list(value.get('e')) if value.get('e') else [None]
```

Comment:
```suggestion
    experiments = force_list(value.get('e')) 
```

Reply:
This is intentionally like this. `force_list` will also convert None to empty list. If we do this we are missing cases like https://github.com/inspirehep/inspire-dojson/blob/master/tests/test_cds.py#L1275-L1300

Reply:
Shouldn't then be force_list(value.get('e', accelerator)) ?

Reply:
No, because this line here https://github.com/inspirehep/inspire-dojson/blob/master/inspire_dojson/cds/rules.py#L404 matches this dict here https://github.com/inspirehep/inspire-dojson/blob/master/inspire_dojson/cds/rules.py#L74.

Reply:
oh ok now I get it, thanks!